pkg/plugin/server.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-plugin"
+)
+
+// TODO: move to config folder.
+type PluginConfig struct {
+	Name       string
+	EntryPoint string
+
+	// Optional checksum.
+	Checksum string
+}
+
+func InitValidators(ctx context.Context, configs []PluginConfig) (map[string]Validator, error) {
+	validators := make(map[string]Validator)
+	var merr *multierror.Error
+	for _, c := range configs {
+		// Enable the plugin.
+		pluginClient := plugin.NewClient(&plugin.ClientConfig{
```

Comment:
I highly recommend splitting this file out to be its own PR

These clients need to be killed. Consider returning a `multicloser` https://github.com/abcxyz/pkg/tree/main/multicloser

Reply:
Split into the following two PRs:
- https://github.com/abcxyz/jvs/pull/290
- https://github.com/abcxyz/jvs/pull/292